doc/classes/TextEdit.xml

```diff
@@ -459,6 +459,12 @@
 		<member name="wrap_enabled" type="bool" setter="set_wrap_enabled" getter="is_wrap_enabled" default="false">
 			If [code]true[/code], enables text wrapping when it goes beyond the edge of what is visible.
 		</member>
+		<member name="scroll_vertical" type="float" setter="set_v_scroll" getter="get_v_scroll" default="0.0">
+			Vertical scroll.
```

Comment:
It could be a little more descriptive, like:
> Value of the vertical scroll.

Same for the horizontal.

Reply:
Or, like in ScrollContainer :
> The current horizontal/vertical scroll value.

Reply:
Thanks, makes sense to have them more descriptive, I'll make the change.

Reply:
@malbach @YeldhamDev I've made the change to the documentation to be more descriptive.